version.txt

```diff
@@ -1,2 +1,2 @@
-0.86.0
+0.86.1
```

Comment:
I am not entirely sure how the release plugin work on the release branch, but this might have to be SNAPSHOT. @emanuelez  ?